packages/gatsby-recipes/rollup.config.js

```diff
@@ -23,38 +24,104 @@ function excludeDevTools() {
   }
 }
 
-export default {
-  input: "src/index.js",
-  output: {
-    file: "dist/index.js",
-    format: "cjs",
+export default [
+  {
+    input: {
+      index: `src/index.js`,
+      "graphql-server/server": `src/graphql-server/server.js`
+    },
+    output: {
+      dir: `dist`,
+      entryFileNames: `[name].js`,
+      format: "cjs",
+      sourcemap: true,
+    },
+    plugins: [
+      replace({
+        values: {
+          "process.env.NODE_ENV": JSON.stringify(`production`)
+        }
+      }),
+      excludeDevTools(),
+      json(),
+      babel({
+        babelHelpers: `runtime`,
```

Comment:
If we keep runtime we'll have to add @babel/runtime as a dependency
```suggestion
        babelHelpers: `bundled`,
```